cli/ast.rs

```diff
@@ -464,10 +466,11 @@ pub fn transpile_module(
       legacy: true,
       emit_metadata: emit_options.emit_metadata
     }),
+    helpers::inject_helpers(),
```

Comment:
Shouldn't this be optional depending on if decorators are enabled or not?

Reply:
We have decorators enabled when transpiling, and they are not configurable off.  The only thing that is configurable is emitting the decorator meta data.  This is because basically if you don't use them there is a _zero_ cost.  Just like setting `"jsx"` to `"react"`.  If you don't use JSX, it doesn't mean anything.

In addition, the helpers only get injected if the code depends upon them.  There are the decorator ones, but if someone were to target < esnext, there would be other helpers required, so we need them on anyways.

Reply:
> We have decorators enabled when transpiling, and they are not configurable off.

This kind of bothers me. I would rather legacy decorators were a config option (or a flag). But SGTM. Makes sense now.